components/Quote/Quote.tsx

```diff
@@ -0,0 +1,36 @@
+import styles from './Quote.module.css';
+
+/**
+ * This is a styled component.
+ *
+ * The WordPress block will have content (passed as innerHTML):
+ *      <p>Before software can be reusable, it first has to be usable.</p><cite>Ralph Johnson</cite>
+ * And a className attribute that is either is-style-default or is-style-large
+ *
+ * The PostContent.tsx will set the large prop to the className and deliver the innerHTML as is.
+ *
+ * The Quote component is output as:
+ *      <blockquote>
+ *          <p>Before software can be reusable, it first has to be usable.</p><cite>Ralph Johnson</cite>
+ *      </blockquote>
+ *
+ * The className varies, and refers to the imported styles in Quote.module.css
+ *
+ * Quote.module.css contains two different styling options - large or default.
```

Comment:
Could this change? Maybe link to its definition in Gutenberg core?

Reply:
Done